docs/guide/exe_verilator.rst

```diff
@@ -1113,6 +1113,12 @@ Summary:
    marking only those signals that need public_flat_rw is typically
    significantly better performing.
 
+.. option:: --public-depth <level>
+
+   Enables public like :vlopt:`--public-flat-rw`, but only to the specified module depth.
+   It still operates at the module level, so under --public-depth 2 given A.B, A.X.B's signals
+    will also be public.
```

Comment:
```suggestion
   Enables public as with :vlopt:`--public-flat-rw`, but only to the specified depth of modules.
   It operates at the module maximum level, so if a module's cells are A.B.X and A.X, the
   a --public-depth 3 must be used to make module X public, and both A.B.X and A.X will be public.
```

Reply:
Oh I understood this wrong, --public-depth 2 wouldn't work here? The maximum level needs to encompass all instances to make the shallow one public?

Reply:
Yes, because level() is the maximum level as we discussed above. To make it work more intuitively you need a different calculation than that used for level().